homeassistant/components/mobile_app/webhook.py

```diff
@@ -538,3 +543,35 @@ async def webhook_get_config(hass, config_entry, data):
         pass
 
     return webhook_response(resp, registration=config_entry.data)
+
+
+@WEBHOOK_COMMANDS.register("get_entities_by_area")
+async def webhook_get_entities_by_area(hass, config_entry, data):
+    """Handle a get entities grouped by area webhook."""
+    area_to_entity_map = {}
+    resp = []
+
+    area_registry = await ar.async_get_registry(hass)
+    device_registry = await dr.async_get_registry(hass)
+    entity_registry = await er.async_get_registry(hass)
+
+    entities = entity_registry.entities
+
+    for entity in entities.values():
+        if not entity.device_id:
+            continue
+        device = device_registry.async_get(entity.device_id)
+        if not device.area_id:
+            continue
+        if device.area_id not in area_to_entity_map:
+            area_to_entity_map[device.area_id] = list()
+        area_to_entity_map[device.area_id].append(entity.entity_id)
+
+    for area_id, entities in area_to_entity_map.items():
+        area = area_registry.async_get_area(area_id)
+
+        area_obj = {"id": area.id, "name": area.name, "entities": entities}
```

Comment:
```suggestion
        area_obj = {"id": area.id, "name": area.name, "entity_ids": entities}
```

 Should we call these `entity_ids`